sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -312,12 +313,16 @@ def call(self):
         break
       except Exception as e:
         self._retry_count += 1
-        logging.info(
-            'Exception at bundle %r, due to an exception: %s',
+        logging.error(
+            'Exception at bundle %r, due to an exception.\n %s',
             self._input_bundle, traceback.format_exc())
         if self._retry_count == self._max_retries_per_bundle:
           logging.error('Giving up after %s attempts.',
                         self._max_retries_per_bundle)
+          if self._retry_count == 1:
+            logging.warning('To apply bundle retry up to %s times, use flag'
```

Comment:
<!--new_thread; commit:593b6f6659973b2774659a66e9be613c535c3f16; resolved:0-->
Let's use `%d` since it is an integer.  This should be `logging.info`.

"Use the experimental flag --direct_runner_bundle_retry to retry failed bundles (up to %d times)."